### PR TITLE
[new] using read-only replica

### PR DESCRIPTION
**Pull request by Rajakavitha1**
# Description

moved the content in [KB](https://app.intercom.com/a/apps/uvcadzno/articles/articles/3235786/show) to the docs.

# Links

Fixes #[insert issue link, if any]

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/timescaledb/latest/contribute-to-docs/)

# Review checklists
Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

- [ ] Is the content technically accurate?
- [ ] Is the content complete?
- [ ] Is the content presented in a logical order?
- [ ] Does the content use appropriate names for features and products?
- [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

- [ ] Is the content free from typos?
- [ ] Does the content use plain English?
- [ ] Does the content contain clear sections for concepts, tasks, and references?
- [ ] Have any images been uploaded to the correct location, and are resolvable?
- [ ] If the page index was updated, are redirects required
      and have they been implemented?
- [ ] Have you checked the built version of this content?

### CODE REVIEW

File: mst/postgresql-read-replica.md [hunk 1]
```diff
@@ -10,6 +10,8 @@ PostgreSQL read-only replicas allow you to perform read-only queries against
 the replica and reduce the load on the primary server. You can optimize query
 response times across different geographical locations because the replica can
 be created in different regions or on different cloud providers.
+For information about creating a read-only replica using aiven client,
```
Comment (Loquacity) [inline mst/postgresql-read-replica.md:13]:
```suggestion
For information about creating a read-only replica using the Aiven client,
```

File: mst/postgresql-read-replica.md [hunk 2]
```diff
@@ -38,3 +39,37 @@ When the read-only replica is created it is listed as a service in your
...
 click the `Promote to master` button.
+
+<procedure>
+
+## Using read-only replica for the service on MST
+
+1.  In the `Overview` page of the read-only replica for the service on MST, copy
+    the `Service URI`.
+
+1.  Connect to the read-only service using psql client:
```
Comment (Loquacity) [inline mst/postgresql-read-replica.md:50]:
```suggestion
1.  At the psql prompt, connect to the read-only service:
```

File: mst/postgresql-read-replica.md [hunk 3]
```diff
@@ -38,3 +39,37 @@ When the read-only replica is created it is listed as a service in your
...
+
+    ```sql
+       psql <SERVICE_URI>
```
Comment (Loquacity) [inline mst/postgresql-read-replica.md:53]: Line the command up with the three backticks

File: mst/postgresql-read-replica.md [hunk 4]
```diff
@@ -38,3 +39,37 @@ When the read-only replica is created it is listed as a service in your
...
+    ```
+
+1.  To check whether you are connected to a primary or replica node:
+
+    ```sql
+     SELECT * FROM pg_is_in_recovery();
```
Comment (Loquacity) [inline mst/postgresql-read-replica.md:59]: Same here

File: mst/postgresql-read-replica.md [hunk 5]
```diff
@@ -38,3 +39,37 @@ When the read-only replica is created it is listed as a service in your
...
+    ```
+
+If the output is `TRUE` you are connected to the replica, and if the output is
+`FALSE` if you are connected to the primary server.
```
Comment (Loquacity) [inline mst/postgresql-read-replica.md:63]:
```suggestion
`FALSE` you are connected to the primary server.
```

File: mst/postgresql-read-replica.md [hunk 6]
```diff
@@ -38,3 +39,37 @@ When the read-only replica is created it is listed as a service in your
...
+       psql <SERVICE_URI>
+    ```
+
+1.  To check whether you are connected to a primary or replica node:
+
+    ```sql
+     SELECT * FROM pg_is_in_recovery();
+    ```
+
+If the output is `TRUE` you are connected to the replica, and if the output is
```
Comment (Loquacity) [inline mst/postgresql-read-replica.md:62]: Line this up with the previous para ("To check whether ...")

File: mst/postgresql-read-replica.md [hunk 7]
```diff
@@ -38,3 +39,37 @@ When the read-only replica is created it is listed as a service in your
...
+`FALSE` if you are connected to the primary server.
+
+</procedure>
+
+<highlight type="note">
+Managed Service for TimescaleDB uses asynchronous replication and hence a lag is
```
Comment (Loquacity) [inline mst/postgresql-read-replica.md:68]:
```suggestion
Managed Service for TimescaleDB uses asynchronous replication, so some lag is
```

File: mst/postgresql-read-replica.md [hunk 8]
```diff
@@ -38,3 +39,37 @@ When the read-only replica is created it is listed as a service in your
...
+expected. When running an `INSERT` operation on the primary node, a minimal
```
Comment (Loquacity) [inline mst/postgresql-read-replica.md:69]:
```suggestion
expected. When you run an `INSERT` operation on the primary node, a small
```

File: mst/postgresql-read-replica.md (hunk 3 above)
Comment (Rajakavitha1) [reply mst/postgresql-read-replica.md]: done

**Review by Loquacity — APPROVED** [review]

**Review by charislam — APPROVED** [review]

**Review by jfjoly — APPROVED** [review]
LGTM

**Pull request merged**